src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -372,6 +378,40 @@ private static void verifyConfig(Configuration testConfig, Configuration parsedC
         }
     }
 
+    /**
+     * Performs verification of violation lines.
+     *
+     * @param config parsed config.
+     * @param file file path.
+     * @param inputConfiguration InputConfiguration object.
+     * @throws Exception if exception occurs during verification process.
+     */
+    private void verifyViolations(Configuration config,
+                                  String file,
+                                  InputConfiguration inputConfiguration) throws Exception {
+        final List<File> files = Collections.singletonList(new File(file));
+        final Checker checker = createChecker(config);
+        final Map<String, List<String>> actualViolations =
+                getActualViolations(checker.process(files));
+        checker.destroy();
+        final List<Integer> expectedViolationLines;
+        if (actualViolations.containsKey(file)) {
+            expectedViolationLines =
+                    actualViolations.get(file)
```

Comment:
you can specify default here `actualViolations.getOrDefault(file, new ArrayList)`, so that if condition can be removed, what do you think?

Reply:
You're right. I should have thought of this.
Done.